### ISSUE TITLE
Time tracking

### ISSUE THREAD

**Issue by Imme1992** [WAM2layers/WAM2layers#9]
There is also an option to track the time until a moisture source is located. That is currently not used in the code, but it would be nice to include this again as well as an option.

**Comment by Peter9192** [comment]
Was there a previous version of the code where this is available? Can you link to that?

**Comment by Imme1992** [comment]
Yes the original code or Ruud has this component in it: https://github.com/ruudvdent/WAM2layersPython/blob/master/Con_E_Recyc_Masterscript.py

**Comment by Peter9192** [comment]
This one, right? https://github.com/ruudvdent/WAM2layersPython/blob/80fe6f11b67144ded278c24b883c965659fe3388/Con_E_Recyc_Masterscript.py#L332

**Comment by ruudvdent** [comment]
will be done when backtracking script is considered 'finished'
